docs/testing/integration-tests.md

```diff
@@ -0,0 +1,131 @@
+# Integration Tests
+
+FPS enables the simulation of proposals within integration tests. This
+capability is essential for verifying the functionality of your proposals and
+ensuring they don't break existing features.
+
+## Setting Up PostProposalCheck.sol
+
+The first step is to create a `PostProposalCheck.sol` contract, which serves as
+a base for your integration test contracts. This contract is responsable for
+deploying proposal contracts, executing them, and updating the address objects. We'll illustrate this with the Multisig example from our [Multisig Proposal Guide](../guides/multisig-proposal.md).
+
+```solidity
```

Comment:
What do we think about moving these code examples into the `examples/` directory?